mmv1/products/apigee/api.yaml

```diff
@@ -660,3 +660,52 @@ objects:
         'Provisioning NAT IPs':
           'https://cloud.google.com/apigee/docs/api-platform/security/nat-provisioning'
       api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances.natAddresses'
+  - !ruby/object:Api::Resource
+    name: 'SyncAuthorization'
+    self_link: 'organizations/{{name}}:getSyncAuthorization'
+    base_url: ''
+    create_url: 'organizations/{{name}}:setSyncAuthorization'
+    update_url: 'organizations/{{name}}:setSyncAuthorization'
```

Comment:
This resource has no default method so there is no effect in the deletion scenario?

Should we not disassociate the authorization upon deletion?
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6912-old..auto-pr-6912#diff-0cc78cc154eefd0a0df135386b0c98eac495dc044525db9154b9f8c63c51d0a7R213-R220

Reply:
This resource has some kind of effect.

setSyncAuthorization will do something. In deletion we should undo the resources action otherwise this resource is not behaving like other resources.

Reply:
setSyncAuthorization will grant permissions to a list of service accounts to allow the Synchronizer to download environment data from the control plane.control plane resources (https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations/setSyncAuthorization). 

I do not find the doc to cancel the permissions.